compiler/rustc_codegen_llvm/src/coverageinfo/mod.rs

```diff
@@ -284,10 +285,14 @@ pub(crate) fn save_cov_data_to_mod<'ll, 'tcx>(
     cx: &CodegenCx<'ll, 'tcx>,
     cov_data_val: &'ll llvm::Value,
 ) {
-    let covmap_var_name = llvm::build_string(|s| unsafe {
-        llvm::LLVMRustCoverageWriteMappingVarNameToString(s);
-    })
-    .expect("Rust Coverage Mapping var name failed UTF-8 conversion");
+    let covmap_var_name = CString::new(
+        // FIXME: wait, we're building a cstring out of a string? huh?
+        llvm::build_string(|s| unsafe {
```

Comment:
This can just be `llvm::build_byte_buffer` instead of `llvm::build_string`; there's no reason to go via an intermediate String at all. I meant to clean this up ages ago but forgot about it.

Reply:
Thanks! I figured you'd probably figured out what this should be already.